rocAL/rocAL_pybind/setup.py

```diff
@@ -18,72 +18,29 @@
+from setuptools.dist import Distribution
 import sys
-import subprocess
 
-from setuptools import setup, Extension, find_packages
-from setuptools.command.build_ext import build_ext
+if sys.version_info < (3, 0):
+    sys.exit('rocal Python Package requires Python > 3.0')
 
+class BinaryDistribution(Distribution):
+    """Distribution which always forces a binary package with platform name"""
+    @classmethod
+    def has_ext_modules(self):
+        return True
 
-class CMakeExtension(Extension):
-    def __init__(self, name, sourcedir=''):
-        Extension.__init__(self, name, sources=[])
-        self.sourcedir = os.path.abspath(sourcedir)
-        if "--backend_hip" in sys.argv:
-            self.backend = "HIP"
-            sys.argv.remove("--backend_hip")
-        elif "--backend_ocl" in sys.argv:
-            self.backend = "OCL"
-            sys.argv.remove("--backend_ocl")
-
-
-class CMakeBuild(build_ext):
-    def run(self):
-        try:
-            subprocess.check_output(['cmake', '--version'])
-        except OSError:
-            raise RuntimeError("CMake must be installed to build the following extensions: " +
-                               ", ".join(e.name for e in self.extensions))
-
-        for ext in self.extensions:
-            self.build_extension(ext)
-
-    def build_extension(self, ext):
-        extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
-        if ext.backend == "HIP" :
-            cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
-                      '-DPYTHON_EXECUTABLE=' + sys.executable, '-DGPU_SUPPORT=ON', '-DBACKEND=HIP']
-        else:
-            cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
-                      '-DPYTHON_EXECUTABLE=' + sys.executable, '-DGPU_SUPPORT=ON', '-DBACKEND=OPENCL']
-
-        cfg = 'Debug' if self.debug else 'Release'
-        build_args = ['--config', cfg]
-
-        cmake_args += ['-DCMAKE_BUILD_TYPE=' + cfg]
-        build_args += ['--', '-j2']
-
-        env = os.environ.copy()
-        env['CXXFLAGS'] = '{} -DVERSION_INFO=\\"{}\\"'.format(env.get('CXXFLAGS', ''),
-                                                              self.distribution.get_version())
-        if not os.path.exists(self.build_temp):
-            os.makedirs(self.build_temp)
-        subprocess.check_call(['cmake', ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
-        subprocess.check_call(['cmake', '--build', '.'] + build_args, cwd=self.build_temp)
-
-setup(name='amd-rocal',
-      description='AMD ROCAL',
-      url='https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/tree/master/rocal',
-      version='1.1.0',
+setup(
+      name='amd-rocal',
+      description='AMD ROCm Augmentation Library',
+      url='https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/rocAL',
+      version='1.0.0',
       author='AMD',
       license='Apache License 2.0',
```

Comment:
do we want to say apache Licence since others are MIT

Reply:
Since PR destination branch was changed to master which didn't want the pybind changes, this was showing up as files changed. The PR destination branch is changed to develop and merged with latest changes.